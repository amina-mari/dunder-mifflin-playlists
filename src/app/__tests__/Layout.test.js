import renderer from 'react-test-renderer';
import Layout from '../layout';

it('render correctly when children is passed', () => {
    const component = renderer.create(<Layout></Layout>)
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();

    
}) 