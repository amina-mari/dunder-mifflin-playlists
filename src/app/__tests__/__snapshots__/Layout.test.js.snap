// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly when children is passed 1`] = `
<html
  className="className"
  lang="pt-br"
>
  <body
    className="main"
  >
    <main
      className="main-container"
    />
    <nav
      className="navigation-dots"
    >
      <div
        className="nav-dot nav-dot_login"
        data-testid="loginDot"
      />
      <div
        className="nav-dot nav-dot_songs"
        data-testid="songsDot"
      />
    </nav>
  </body>
</html>
`;
